integration/update_test.go

```diff
@@ -770,6 +770,66 @@ func TestUpdateMany(t *testing.T) {
 	}
 }
 
+func TestMultiFlag(t *testing.T) {
+	t.Parallel()
+
+	for name, tc := range map[string]struct {
+		filter bson.D
+		update bson.D
+		multi  string
```

Comment:
```suggestion
		multi  bool
```

Reply:
@fcoury I'm not really sure why this parameter is needed at all
Shouldn't all the tests in the suite run the `update` command with `multi` set to `true`?